### PR TITLE
Use Cobra CLI library, reorganize a bit.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	"github.com/dlorenc/scorecard/checker"
+	"github.com/dlorenc/scorecard/checks"
+	"github.com/dlorenc/scorecard/roundtripper"
+	"github.com/google/go-github/v32/github"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// var repo = flag.String("repo", "", "url to the repo")
```
Comment (inferno-chromium) [inline cmd/root.go:22]: Should these comments be removed ?

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+var rootCmd = &cobra.Command{
+	Use:   "hugo",
+	Short: "Hugo is a very fast static site generator",
+	Long: `A Fast and Flexible Static Site Generator built with
+				  love by spf13 and friends in Go.
+				  Complete documentation is available at http://hugo.spf13.com`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg := zap.NewProductionConfig()
+		// cfg.Level.SetLevel(*logLevel)
```
Comment (inferno-chromium) [inline cmd/root.go:39]: Should this be uncommented.

**Review by inferno-chromium — APPROVED** [review]

File: cmd/root.go (hunk 1 above)
Comment (dlorenc) [reply cmd/root.go:22]: Fixed!

File: cmd/root.go (hunk 2 above)
Comment (dlorenc) [reply cmd/root.go:39]: Whoops, fixed!

**Pull request merged**